index.html

```diff
@@ -4545,7 +4545,20 @@ <h2>
                 <li>Set <var>request</var>.<a>[[\state]]</a> to
                 "<a>closed</a>".
                 </li>
-                <li>Reject the promise
+                <li>If <var>request</var><a>[[\response]]</a> is not null,
+                then:
+                  <ol>
+                    <li>Let <var>response</var> be
+                    <var>request</var><a>[[\response]]</a>.
```

Comment:
Missing dot. Also maybe you should do this before the "if request.[[response]] is not null" step (and then just check if response is null)